src/ui/components/TransferGrain.js

```diff
@@ -25,13 +25,13 @@ export const TransferGrain = ({ledger, setLedger}: Props) => {
   const [maxAmount, setMaxAmount] = useState<Grain>(fromInteger(0));
   const [memo, setMemo] = useState<string>("");
 
-  const setSender = (acct: Account) => {
-    setMaxAmount(acct.balance);
-    setSourceIdentity(acct.identity);
+  const setSender = (acct: Account | null) => {
```

Comment:
Why the change in signature?

Reply:
I felt it was necessary to be able to set the account back to the null default if you cleared the combobox field. This disables the grain transfer button again and prevents unexpected behavior.

Reply:
Cool, makes sense.